django/middleware/csp.py

```diff
@@ -14,23 +12,24 @@ def process_request(self, request):
         request._csp_nonce = LazyNonce()
 
     def process_response(self, request, response):
-        # In DEBUG mode, exclude CSP headers for specific status codes that
-        # trigger the debug view.
-        exempted_status_codes = {
-            HTTPStatus.NOT_FOUND,
-            HTTPStatus.INTERNAL_SERVER_ERROR,
-        }
-        if settings.DEBUG and response.status_code in exempted_status_codes:
-            return response
-
         nonce = get_nonce(request)
-        for header, config in [
-            (CSP.HEADER_ENFORCE, settings.SECURE_CSP),
-            (CSP.HEADER_REPORT_ONLY, settings.SECURE_CSP_REPORT_ONLY),
+        for config, header, disabled in [
```

Comment:
Small nitpick, could we keep the previous order, adding the disabled at the end?
```suggestion
        for header, config, disabled in [
```

Reply:
I switched it to match the order it is being used in the conditional within this loop. This wasn't an arbitrary change I just felt it read better with the 3 conditionals after following the same order. 

That order could be discussed too, but I feel like `config` makes sense first. The `header not in response` and the `not disabled` could be swapped with the assumption that using the decorators would happen more often than a custom header set in the view.

I'm open to however you'd like to tweak these but thought I'd share my thought process.